README.md

```diff
@@ -67,10 +67,11 @@ The following keys are loaded from the keyring on start:
 
 - `peer-key` Ed25519 private key (required)
 - `encryption-key` AES-128, AES-192, or AES-256 key (optional)
+- `identity-key` Secp256k1 private key (optional). This key is used for node's identity.
```

Comment:
Todo: I have mixed feelings about the name identity key, It can be very easily confusing and can be mistaken for requestor's identity. I propose adding the word node in it perhaps.

Reply:
ok, will change to `node-identity-key`